src/material-experimental/mdc-chips/chip-set.ts

```diff
@@ -37,7 +37,7 @@ let uid = 0;
  * @docs-private
  */
 class MatChipSetBase {
-  disabled!: boolean;
+  get disabled(): boolean { return false; }
```

Comment:
Why is this a readonly property now? Maybe we should change the base class to be abstract instead?

Reply:
Yeah, I asked the same above for `MatRadioButtonBase`. I agree that this would be ideal.